### DISCUSSION TITLE
Docker version?

### DISCUSSION THREAD

**Discussion by markuswells** [adriankumpf/tesla_auth#70] [General]
Is there any chance of getting this in a docker container?
I would like to be able to install the container, connect to it and get the tokens.
Then remove the entire docker container.

**Comment by 170031284** [comment]
Hi Mark , 

It is possible , Docker itself is most portable and disposable . 
Step 1: Run chatwoot in docker compose .
step2: generate the tokens 
step3: use docker-compose down to dispose the docker image 

But you should be very careful when your are dealing with production environments and keep in mind about the datalose and privacy
